luigi/task.py

```diff
@@ -128,6 +128,10 @@ class MyTask(luigi.Task):
     #: Defaults to 0 or value in luigi.cfg
     worker_timeout = None
 
+    #: Owner of the task: it is used for sending out error email in addition to
+    #: the one defined in luigi config
+    owner = None
```

Comment:
I would either have something like `owner_email = None` or let `owner` be some sort of class `luigi.TaskOwner` with fields like `email` and such. Then organizations can add whatever fields they want to their `TaskOwner` subclass class.